spec/parsers/conda_spec.rb

```diff
@@ -17,116 +17,98 @@
           {:name=>"ncurses", :requirement=>"6.1", :type=>"runtime"},
           {:name=>"numpy", :requirement=>"1.16.4", :type=>"runtime"},
           {:name=>"openssl", :requirement=>"1.1.1c", :type=>"runtime"},
-          {:name=>"pip", :requirement=>"19.2.2", :type=>"runtime"},
```

Comment:
no version because we're not resolving the version now for manifests, just returning as is. this example file has pip and setuptools without versions